File: assign3/src/vehicles/Car.java
package vehicles;

public class Car extends Vehicle{

  private int _numWheels;

  public Car(int capacity){
    super();
    _numWheels = 4;
    setCapacity(capacity);
    setOccupants(0);
  }

  public int getNumWheels() { return _numWheels; }
  public void setNumWheels(int wheels){
    _numWheels = wheels;
  }
}

File: Assignment4/src/Car.cpp
#include "Car.h"

namespace vehicles{
  Car::Car(int capacity){
    _numWheels = 4;
    setCapacity(capacity);
    setOccupants(0);
  }
  int Car::getNumWheels() const { return _numWheels; }
  void Car::setNumWheels(int wheels){
    _numWheels = wheels;
  }
}
